SCMS.Services.Api/Brokers/Storages/StorageBroker.Student.cs

```diff
@@ -25,7 +27,15 @@ public async ValueTask<Student> InsertStudentAsync(Student student)
 
             return studentEntityEntry.Entity;
         }
+      
+        public async ValueTask<Student> SelectStudentByIdAsync(Guid studentId)
+        {
+            using var broker = new StorageBroker(this.configuration);
+            broker.ChangeTracker.QueryTrackingBehavior = QueryTrackingBehavior.NoTracking;
 
+            return await broker.Students.FindAsync(studentId);
+        }
+      
         public IQueryable<Student> SelectAllStudents() => this.Students.AsQueryable();
```

Comment:
Select All should be implemented first then followed by SelectById

Reply:
Thank you for you reviews.